File: frontend/src/app/graph/RankingResponse.ts
export class RankingResponse {
  minTime: number;
  maxTime: number;
  averageTime: number;
  minScore: number;
  maxScore: number;
  averageScore: number;

  constructor(minTime: number, maxTime: number, averageTime: number, minScore: number, maxScore: number, averageScore: number) {
    this.minTime = minTime;
    this.maxTime = maxTime;
    this.averageTime = averageTime;
    this.minScore = minScore;
    this.maxScore = maxScore;
    this.averageScore = averageScore;
  }
}

File: backend/src/main/java/com/graphs/entity/RankingResponse.java
package com.graphs.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RankingResponse {
    private double minTime;
    private double maxTime;
    private double averageTime;
    private double minScore;
    private double maxScore;
    private double averageScore;
}
